BDArmory/Distribution/GameData/BDArmory/ChangeLog.txt

```diff
@@ -49,20 +76,36 @@
 			- Improvements to regular missile orbital guidance, missiles can now use homingType = orbital without RCS.
 			- Missiles with RCS will use RCS to offset gravity.
 			- Add targetCoM configuration option for IR missiles. If set to true, the missile will target the center of mass instead of the hottest part.
+			- Firing conditions for IR missiles tightened up, AI should no longer select rear-Aspect IR missiles against head-on targets.
+			- Missiles with RCS can now have multiple forwardRCS transforms.
+			- Adds new ModuleMissileMagazine, MultiMissileLauncher reloadable rails and missile pods can now resupply from external missile magazine parts.
 		- Fixes
 			- Fix Modular Missiles with Orbital Guidance not hitting vessel and missile targets.
 			- Fix point defense missile interceptors not firing.
 			- Missile turrets now track targets properly in orbit.
 			- Fix missiles with no targeting type (i.e. AIR-2) not firing within engage range.
+			- Fix IR missiles occasionally suddenly switching lock to allied craft behind them.
+			- Fix edge-case issue where firing radar missiles would get soft-locked if trying to switch to a radar lock that didn't exist.
+			- Fix missile exhaust plume FX scaling.
+			- Fix missile softlock issue if AI tries to fire a laser/GPS missile at a target further than the targeting camera's max range.
+			- Fix nukes in vacuum erroneously using the wrong mass value for calculating impulse from the detonation.
 		- Parts / Balancing
 			- Add new Kinetic Kill Vehicle (KKV) missile. The KKV is an orbital missile with 6 km/s delta-V and no warhead. It relies on kinetic energy from hitting a target directly to deal damage.
 			- Re-balanced the HE-KV-1 missile. The HE-KV-1 now has 3 km/s delta-V with a 60 km range and is best at shorter ranges (under 30 km). The HE-KV-1 retains the same mass.
 			- Orbital guidance missiles now have drag applied to them in atmosphere.	
+	- Gun accuracy improved.
```

Comment:
Maybe expand this a bit to make it clear that it's the guns' deviations that changed, not improvements to the AI's aiming accuracy.

Should we include a brief message as to why they're being improved?